doc/dev/developer_guide.md

```diff
@@ -37,7 +37,7 @@ $ make test
 ```
 
 For more information on running testing and correctly configuring your environment,
-refer to the [`Running the Tests Locally`][running-the-tests] document.
+refer to the [`Running the Tests Locally`][running-the-tests] document. Also, to check hot to test the changes performed locally see the [`Testing changes Locally`][testing-changes-locally] document.
 
```

Comment:
To split the info and we have a doc describing how to test locally under the Testing category.